docs/source/optim.rst

```diff
@@ -155,26 +161,20 @@ should write your code this way:
   if you are calling ``scheduler.step()`` at the wrong time.
 
 
-.. autoclass:: torch.optim.lr_scheduler.LambdaLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.MultiplicativeLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.StepLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.MultiStepLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.ExponentialLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.CosineAnnealingLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.ReduceLROnPlateau
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.CyclicLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.OneCycleLR
-    :members:
-.. autoclass:: torch.optim.lr_scheduler.CosineAnnealingWarmRestarts
-    :members:
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    lr_scheduler.LambdaLR
+    lr_scheduler.MultiplicativeLR
+    lr_scheduler.StepLR
+    lr_scheduler.MultiStepLR
+    lr_scheduler.ExponentialLR
+    lr_scheduler.CosineAnnealingLR
+    lr_scheduler.ReduceLROnPlateau
+    lr_scheduler.CyclicLR
+    lr_scheduler.OneCycleLR
+    lr_scheduler.CosineAnnealingWarmRestarts
```

Comment:
This one read a little weirdly:
![image](https://user-images.githubusercontent.com/5652049/114427893-8d658e80-9b89-11eb-8bec-126fff634008.png)